README.md

```diff
@@ -131,6 +131,14 @@ Usage:
 ./hack/machineset.sh apply/delete    # to create/delete MachineSet directly on cluster
 ```
 
+## Applying autoscaling to WMCO-managed Windows instances
+
+Cluster autoscaling is supported for Windows instances. 
+
+- Define and deploy a [ClusterAutoscaler](https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html#configuring-clusterautoscaler).
+- Ensure a MachineSet resource is deployed for Windows machines (see spec in [Usage section](https://github.com/openshift/windows-machine-config-operator#usage)).
```

Comment:
I wanted to emphasize that autoscaling requires a MachineSet be present. I will change this to `Create a Windows node through a MachineSet` and that way it will stay relevant when BYOH support is introduced.

Reply:
I am fine with that.